common/common.go

```diff
@@ -219,6 +219,8 @@ const (
 	EnvVarTLSDataPath = "ARGOCD_TLS_DATA_PATH"
 	// EnvGitAttemptsCount specifies number of git remote operations attempts count
 	EnvGitAttemptsCount = "ARGOCD_GIT_ATTEMPTS_COUNT"
+	// EnvGitAutopackLimit specifies the limit for automatic git repacking
+	EnvGitAutopackLimit = "ARGOCD_GIT_AUTOPACK_LIMIT"
```

Comment:
could you please also provide it as option to pass from config map?

Reply:
Make a lot of sense! For sure :)